crates/ln-dlc-node/src/node.rs

```diff
@@ -573,7 +575,7 @@ impl Node {
                     base_msat: 1000,
                     proportional_millionths: LIQUIDITY_ROUTING_FEE_MILLIONTHS,
                 },
-                cltv_expiry_delta: MIN_CLTV_EXPIRY_DELTA,
+                cltv_expiry_delta: 144,
```

Comment:
Can we add a comment why this needs to be `144` please?

Reply:
Actually it is also working with MIN_CLTV_EXPIRY_DELTA (42). We've changed it in some previous attempts as this was the setting used by Breez and Phoenix.